Packs/PagerDuty/Integrations/PagerDuty/PageDuty_test.py

```diff
@@ -41,3 +42,278 @@ def test_get_incidents(requests_mock, mocker):
+                                    },
+                                    "incident": {
+                                        "id": "PXP12GZ",
+                                        "type": "incident_reference",
+                                        "summary": "[#30990] Test Incident for Demisto Integration - No Action",
+                                        "self": "https://api.pagerduty.com/incidents/PXP12GZ",
+                                        "html_url": "https://upstart.pagerduty.com/incidents/PXP12GZ"
+                                    },
+                                    "updated_at": "2022-02-24T18:08:50-08:00",
+                                    "message": "Please help with issue - join bridge at +1(234)-567-8910",
+                                    "requester": {
+                                        "id": "P09TT3C",
+                                        "type": "user_reference",
+                                        "summary": "John Doe",
+                                        "self": "https://api.pagerduty.com/users/P09TT3C",
+                                        "html_url": "https://mycompany.pagerduty.com/users/P09TT3C",
+                                        "avatar_url": "https://secure.gravatar.com/avatar/\
+                                        1c747247b75acc1f724e2784c838b3f8.png?d=mm&r=PG",
+                                        "job_title": ""
+                                    },
+                                    "requested_at": "2022-02-25T00:58:27Z"
+                                }
+                            ]
+                        }
+                    }
+                ]
+            }
+        }
+    )
+
+    from PagerDuty import add_responders_to_incident
+    res = add_responders_to_incident(**demisto.args())
+    expected_users_requested = ','.join([x.get("ID") for x in res.outputs])
+    assert demisto.args().get('incident_id') == res.outputs[0].get('IncidentID')
+    assert demisto.args().get('message') == res.outputs[0].get('Message')
+    assert demisto.params().get('DefaultRequestor') == res.outputs[1].get('RequesterID')
+    assert demisto.args().get('user_requests') == expected_users_requested
+
+
+def test_add_responders_default(requests_mock, mocker):
+    """
+    Test sending request to a responder without specifying responders
+    """
+    mocker.patch.object(
+        demisto,
+        'params',
+        return_value={
+            'APIKey': 'API_KEY',
+            'ServiceKey': 'SERVICE_KEY',
+            'FetchInterval': 'FETCH_INTERVAL',
+            'DefaultRequestor': 'P09TT3C'
+        }
+    )
+    mocker.patch.object(
+        demisto,
+        'args',
+        return_value={
+            "incident_id": "PXP12GZ",
+            "message": "Please help with issue - join bridge at +1(234)-567-8910"
+        }
+    )
+    requests_mock.post(
+        'https://api.pagerduty.com/incidents/PXP12GZ/responder_requests',
+        json={
```

Comment:
please put all the json mock response in a file, it would make the unit test more readable